### ISSUE TITLE
Erroneous missing parameter name using create_hit and layout_params

### ISSUE THREAD

**Issue by Langholt** [boto/boto#3056]
I'm trying to create a hit with a HIT layout ID. This is successful if no parameters are needed for the layout. If I try passing a single parameter using layout_parms, create_hit returns an error. This happens even if the name of the parameter is right. Code an resulting error to follow 

An example code is as follows:
mtc = MTurkConnection(aws_access_key_id=AWSKEY,
                      aws_secret_access_key=AWSSKEY,
                      host=HOST)
#Set the layoutID
LAYOUTID = '39ZKK3W0VMKU0DPHRLCXKSX4INR56I' #for sandbox
#Set the url to pass a a paramtere into layout
S3URL = LayoutParameter('image_url', 'https://s3.amazonaws.com/'+BUCKET_NAME+'/'+SERIAL)
REWARD = 0.05
#Create hit in Mturk
    hit_result = mtc.create_hit(hit_layout=LAYOUTID,layout_params=S3URL,  keywords=keywords_list, reward=REWARD,duration=datetime.timedelta(7),max_assignments=1)

Error:
Your request is missing required parameters. Required parameters include HITLayoutParameter. You have not provided all required HITLayout parameters. Missing parameter names: image_url

**Comment by Bustead** [comment]
@Langholt did you find a fix for this?

**Comment by Langholt** [comment]
@Bustead Unfortunately not (at least not one that I can remember).

**Comment by Bustead** [comment]
Can you reproduce the issue in Boto3? I'm happy to go and investigate and see if I can reproduce, but since this issue is so old I'm wondering if it's worth the effort!

**Comment by Langholt** [comment]
Nope. I stopped exploring this a long time go. I wouldn't waste time. If it hasn't been fixed now it probably never will.

**Comment by Bustead** [comment]
Thanks @Langholt , wanna close the issue then? I know it's not resolved but if it's gone stale and we aren't chasing it, we might as well close it out imo.
